File: 프로그래머스/0/120806. 두 수의 나눗셈/두 수의 나눗셈.py
import math 

def solution(num1, num2):
    return math.trunc(num1 / num2 * 1000)

# math.trunc(숫자)
# 반올림 하지 않고, 소수 부분만 제거
# 숫자 아니면, TypeError 발생

File: 프로그래머스/0/120806. 두 수의 나눗셈/두 수의 나눗셈.java

class Solution {
    public int solution(int num1, int num2) {
        double divide = (double) num1 / num2; // double과 int 연산 시 double로 자동 형변환 수행
        return (int) Math.floor(divide * 1000); 
    }
}

// Math.floor(double a)
// 소수점 내림 함수

// 방법 2
// (int)((double) num1 / num2 * 1000)
// 1. (double) num1 / num2 * 1000 -> 값
// 2. (int) 값 -> 소수점 이하 손실